Volume/Views/UnderlinedText.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UnderlinedText.swift
+//  Volume
+//
+//  Created by Daniel Vebman on 11/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct UnderlinedText: View {
+    
+    @State var text: String
+    @State private var textSize: CGSize = .zero
+    
+    init(_ text: String) {
+        _text = State(initialValue: text)
+    }
+
```

Comment:
Seems unnecessary here and in other places, as it just allows us to leave out the `text: text` when creating the `UnderlineView` if I'm understanding it correctly.

Reply:
You're right, it isn't necessary; it's purely aesthetic. I did it like this to mimic SwiftUI's native `Text`, which omits the parameter name.